app/src/main/java/ai/elimu/analytics/db/RoomDb.java

```diff
@@ -44,7 +46,7 @@
         StoryBookLearningEvent.class,
 
         VideoLearningEvent.class
-})
+}, autoMigrations = {@AutoMigration(from = 22, to = 23, spec = AutoMigrationSpecFrom22To23.class)})
```

Comment:
Using auto migration feature reduces the effort of writing custom SQL commands to:
- Create a new table
- Copy old data over the new table
- Drop the old table
- Rename the new table 

even though under the hood the same strategy is being used in Room DB for deleting column(s) from existing table(s).